index.html

```diff
@@ -211,22 +218,34 @@
             title: {
                 display: true,
                 fontSize: 24,
-                text: 'Combined PR Age',
-            },
-            legend: {
-                display: false,
+                text: 'Pull Request Age',
             },
             scales: {
                 xAxes: make_xAxes(timeframe_github),
                 yAxes: [
+                    {
+                        type: 'linear',
+                        display: 'auto',
+                        id: 'leftAxis',
+                        position: 'left',
+                        scaleLabel: {
+                            display: true,
+                            labelString: 'Average Age (days)',
+                        },
+                        ticks: {
+                            min: 0,
+                            max: 80,
+                            stepSize: 10,
+                        },
+                    },
                     {
                         type: 'linear',
                         display: 'auto',
                         id: 'rightAxis',
                         position: 'right',
                         scaleLabel: {
                             display: true,
-                            labelString: 'PR-months',
+                            labelString: 'Combined Age (PR-months)',

```

Comment:
I can't decide if this scale is pessimistic or optimistic XD

Reply:
It's easy to come back later and change to PR-millenia.

Reply:
With ~30 open PRs, we accumulate one PR-month per day. :scream_cat: